examples/hash_content.rs

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env rust-script
+//! This is free and unencumbered software released into the public domain.
+//!
+//! ```cargo
+//! [dependencies]
+//! protoflow = "0.4.3"
+//! ```
+
+use protoflow::{blocks::*, BlockResult};
+
+fn main() -> BlockResult {
+    System::run(|s| {
+        let content = s.const_string("Hello, World!");
+        let line_encoder = s.encode_lines();
+        let hash_content = s.hash_sha2();
+        let stdout = s.write_stdout();
+        s.connect(&content.output, &line_encoder.input);
+        s.connect(&line_encoder.output, &hash_content.input);
+        s.connect(&hash_content.hash, &stdout.input);
```

Comment:
It's currently writing the raw bytes (unprintable/unreadable) of the hash to stdout. Hex encode the hash first:
```suggestion
        let encode_hex = s.encode_hex();
        let stdout = s.write_stdout();
        s.connect(&content.output, &line_encoder.input);
        s.connect(&line_encoder.output, &hash_content.input);
        s.connect(&hash_content.hash, &encode_hex.input);
        s.connect(&encode_hex.output, &stdout.input);
```